src/lib/libast/regex/regcomp.c

```diff
@@ -1410,7 +1415,9 @@ static_fn Rex_t *regcomp_bra(Cenv_t *env) {
                         case 0:
                             goto error;
                         case ':':
-                            if (env->flags & REG_REGEXP) goto complicated_normal;
+                            if (env->flags & REG_REGEXP) {
```

Comment:
It's okay if it remains this way but you could revert this to its original single line form. Also, several places below.